torchft/process_group_test.py

```diff
@@ -178,6 +193,17 @@ def test_baby_gloo(self) -> None:
 
         torch.testing.assert_close(at, bt)
```

Comment:
We can assert an exact value here? Would be more a check that all reduce actually does all reduce which might not be the intention here

Reply:
done